src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -30,8 +30,13 @@ foam.CLASS({
     'foam.u2.TableView'
   ],
 
+  imports: [
+    'window'
+  ],
+
   constants: {
-    CSS_NAME: 'foam-u2-search-FilterController'
+    CSS_NAME: 'foam-u2-search-FilterController',
```

Comment:
So is this a problem with CSS inheritence?  I'm PolymerController and I extend FilterController, but this.myCls() resolves to 'PolymerController' and the CSS classes installed by FilterController don't get applied to my dom nodes?